app/models/loader/types.rb

```diff
@@ -23,14 +23,14 @@ def admin_role account
       # Wraps a policy object with a corresponding +Loader::Types+ object.
       #
       # +external_handler+ should provide the methods +policy_id+, +handle_password+,
-      # +handle_public_key+. This argument is optional if the policy will not use 
+      # +handle_public_key+. This argument is optional if the policy will not use
       # that functionality.
       def wrap obj, external_handler = nil
         cls = Types.const_get obj.class.name.split("::")[-1]
         cls.new obj, external_handler
       end
     end
-    
+
```

Comment:
See note above about cleanup commits vs code change commits. All of these whitespace changes should go in a separate commit.